### PR TITLE
Mavic 2 PRO

### PR DESCRIPTION
**Pull request by fabienrohrer**
- [x] Add Mavic 2 PRO model
    - [x] Add `Mavic2PRO.proto`
    - [x] Add icon
    - [x] Add doc
    - [x] Add robot component + fix it to support the propellers
- [x] Create an example
    - [x] Create a basic controller in C
    - [x] Add doc
- [x] Add the world to the guided tour
- [x] Change log

Doc preview: https://cyberbotics.com/doc/guide/mavic-2-pro?version=feature-mavic-simulation

### CODE REVIEW

**Comment by fabienrohrer** [comment]
First import

mavic_2_pro

**Comment by fabienrohrer** [comment]
The model is more or less completed:
- meshes are done
- joints are set
- propellers and masses are calibrated
- devices are set

ezgif-2-a7e709b39295
mavic_2_pro

Remains now to a do a demo controller.

**Comment by fabienrohrer** [comment]
👍

File: projects/robots/dji/mavic/worlds/mavic_2_pro.wbt [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+TeslaModel3Simple {
+  translation -41.5139 0.4 -4.34169
+  rotation 0 -1 0 -1.3089953071795861
+}
+Pine {
+  translation -14.0083 -1.06581e-14 14.4828
+  rotation 0 1 0 3.14159
+}
+Pine {
+  translation -19.6195 3.55271e-15 24.3783
```
Comment (DavidMansolino) [inline projects/robots/dji/mavic/worlds/mavic_2_pro.wbt]:
```suggestion
  translation -19.6195 0 24.3783
```

File: projects/robots/dji/mavic/worlds/mavic_2_pro.wbt [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+  rotation 0 1 0 2.87979
+  name "pine tree(1)"
+}
+Pine {
+  translation -22.0129 0 -6.20205
+  name "pine tree(3)"
+}
+Pine {
+  translation -10.6926 -1.42109e-14 25.1339
```
Comment (DavidMansolino) [inline projects/robots/dji/mavic/worlds/mavic_2_pro.wbt]:
```suggestion
  translation -10.6926 0 25.1339
```

File: projects/robots/dji/mavic/worlds/mavic_2_pro.wbt [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+  name "pine tree(4)"
+}
+Pine {
+  translation -14.3441 -2.13163e-14 -14.5621
```
Comment (DavidMansolino) [inline projects/robots/dji/mavic/worlds/mavic_2_pro.wbt]:
```suggestion
  translation -14.3441 0 -14.5621
```

File: projects/robots/dji/mavic/worlds/mavic_2_pro.wbt [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+  name "pine tree(5)"
+}
+Pine {
+  translation -26.6328 -2.13163e-14 7.1707
```
Comment (DavidMansolino) [inline projects/robots/dji/mavic/worlds/mavic_2_pro.wbt]:
```suggestion
  translation -26.6328 0 7.1707
```

File: projects/robots/dji/mavic/worlds/mavic_2_pro.wbt [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+  density 0.01
+  type "cherry tree"
+  randomSeed 1
+  maxHeight 12
+  minHeight 6
+  maxRadius 6
+  minRadius 3
+}
+Mavic2Pro {
+  translation 1.87638e-07 0.1 1.76041e-13
```
Comment (DavidMansolino) [inline projects/robots/dji/mavic/worlds/mavic_2_pro.wbt]:
```suggestion
  translation 0 0.1 0
```

**Comment by fabienrohrer** [comment]
@DavidMansolino I think I tool all your comments into consideration.

A quick movie to show the current result:

https://www.youtube.com/watch?v=q8QUEBJl7Qw

[IMAGE ALT TEXT HERE](https://www.youtube.com/watch?v=q8QUEBJl7Qw)

**Comment by fabienrohrer** [comment]
👍

**Pull request merged**